dicoogle/src/main/java/pt/ua/dicoogle/server/users/UserExportPresets.java

```diff
@@ -66,18 +59,25 @@ public static void savePreset(String username, String presetName, String[] field
      * @throws IOException if an I/O error occurs
      */
     public static Map<String, String[]> getPresets(String username) throws IOException {
-        File presetsDir = new File(Platform.homePath() + "users/" + username + "/presets/");
+        Path presetsDir = Paths.get(Platform.homePath(), "userdata", username, "presets");
 
         Map<String, String[]> presets = new HashMap<>();
 
-        if (!presetsDir.isDirectory()) {
+        if (!Files.isDirectory(presetsDir)) {
             return presets;
         }
 
-        for (File presetFile : presetsDir.listFiles()) {
-            String[] fields = FileUtils.readFileToString(presetFile, "UTF-8").split("\n");
-            presets.put(FilenameUtils.removeExtension(presetFile.getName()), fields);
-        }
+        Stream<Path> fileList = Files.list(presetsDir);
+        fileList.filter(file -> !Files.isDirectory(file)).forEach(file -> {
+            try {
+                String[] preset = Files.readAllLines(file).toArray(new String[0]);
+                String fileName = file.getFileName().toString();
+                String presetName = fileName.substring(0, fileName.lastIndexOf('.'));
+                presets.put(presetName, preset);
+            } catch (IOException e) {
+                logger.error("Could not read preset.", e);
```

Comment:
The exception might not include the name of the file.

```suggestion
                logger.error("Could not read preset from `{}`", file.getFileName(), e);
```